tests/D2L.CodeStyle.Analyzers.Test/Specs/ImmutabilityAnalyzer.cs

```diff
@@ -889,12 +889,12 @@ public sealed class AnalyzedImmutableGenericClassGivenT<[ConditionallyImmutable.
 
 
 
-		static Types.SomeImmutableGenericInterfaceRestrictingT<T, U> /* MemberIsNotReadOnly(Field, m_field, AnalyzedImmutableGenericClassGivenT) */ m_field /**/;
-		static readonly Types.SomeImmutableGenericInterfaceRestrictingT<T, U> m_field;
-		Types.SomeImmutableGenericInterfaceRestrictingT<T, U> /* MemberIsNotReadOnly(Field, m_field, AnalyzedImmutableGenericClassGivenT) */ m_field /**/;
-		readonly Types.SomeImmutableGenericInterfaceRestrictingT<T, U> m_field;
-		Types.SomeImmutableGenericInterfaceRestrictingT<T, U> Property { get; }
-		Types.SomeImmutableGenericInterfaceRestrictingT<T, U> Property { get { return default; } }
+		static Types.SomeImmutableGenericInterfaceRestrictingT</* TypeParameterIsNotKnownToBeImmutable(T) */ T /**/, U> /* MemberIsNotReadOnly(Field, m_field, AnalyzedImmutableGenericClassGivenT) */ m_field /**/;

```

Comment:
conditional type parameters aren't considered immutable when checking arguments to immutable type parameters (`<[Immutable] T>`). These parameters are expected definitely immutable things.